packages/editor/src/core/components/Cell/InsertNew.tsx

```diff
@@ -31,6 +31,25 @@ const InsertNew: React.FC<{ parentCellId?: string }> = ({ parentCellId }) => {
     },
   });
 
+  React.useEffect(() => {
+    window?.addEventListener('keydown', handleKeyDown);
+    return () => {
+      window?.removeEventListener('keydown', handleKeyDown);
+    };
+  }, []);
+
+  const handleKeyDown = (e: KeyboardEvent) => {
+    if (e?.key === '/') {
```

Comment:
can we make this configurable? (could also be done later)

Reply:
I didn't get this point throw an example like how you're expecting

Reply:
i am open for a suggestion, but we can omit it for the time being